src/osd/scrubber/pg_scrubber.cc

```diff
@@ -842,7 +842,7 @@ int PgScrubber::get_whoami() const
  * - end
  * - start
  */
-bool PgScrubber::select_range()
+int64_t PgScrubber::select_range()
```

Comment:
The comment immediately above should indicate what this return value is.

Reply:
Why int64_t?  You aren't using a negative value to encode an error (though that would be a valid choice).  It doesn't look like it's possible for this function to return a negative value.

Reply:
(that's an old and highly debated issue in "C++ circles". My preference is to follow the "do not use unsigned to anything else but bitmasks" - as Google says. But there are other views. A contrasting one is here:
https://youtu.be/82jVpEmAEV4?si=QJAjsd4QBXwcJYc5
)

Reply:
I'd *really* rather types encode the actual range they are supposed to represent.